tools/myst/host/package.c

```diff
@@ -603,6 +603,30 @@ int _exec_package(
         }
     }
 
+    /* Get --myst-rootfs=<path> option if any  */
+    {
+        const char* arg = NULL;
+
+        if ((cli_getopt(&argc, argv, "--myst-rootfs", &arg) == 0))
+        {
+            if (access(arg, R_OK) != 0)
+            {
+                fprintf(
+                    stderr, "%s: bad --myst-rootfs option: %s", argv[0], arg);
```

Comment:
Be more specific on the error.

Reply:
Fixed. I turned it into a "file not found" error that mentions the option name.

Reply:
I think the consensus is to group Mystikos-specific command line options before the application specific command line options, but keep the option name of the former without `myst-`.